.github/semantic.yml

```
@@ -12,6 +12,8 @@ types:
   - ci
   - chore
   - build
+  - break
```

dummy change (will revert)

```suggestion
  - break  
```